pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,202 @@
+# Running Buildkite Agent on Google Cloud Platform
+
+The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). For fine control over long–lived agents, you might like to run the agent using individual VM instances on Google Compute Engine. Or run Docker–based builds using a scalable cluster of agents on the Google Container Engine.
+
+<%= toc %>
+
+## Running the agent on Google Compute Engine
+
+To run the agent on your own [Google Compute Engine](https://cloud.google.com/compute) instance use whichever installer matches your instance type. For example:
+
+Launch an instance using the latest Ubuntu LTS image [through the console](https://console.cloud.google.com/compute/instancesAdd#preconfigured-image-ubuntu-1604-xenial-v20170202):
```

Comment:
The most important parts of that are "Launch an instance" and "latest Ubuntu LTS image", the "through the console" is optional and an easy shortcut. More experienced folks will probably already have a way of doing this they prefer, like doing it from a shell. I think I'd prefer to leave it in the current order if that's okay?